play-media-native/screens/CreateEvent/FieldsView.tsx

```diff
@@ -1,12 +1,67 @@
+import { useCallback } from 'react';
+import { Pressable, View } from 'react-native';
+
 import { inputContainerStyle } from './styles';
+import { DatePicker } from '../../components/DatePicker/DatePicker';
 import { InputText } from '../../components/InputText/InputText';
+import { SportPicker } from '../../features/SportPicker/SportPicker';
+import { getDate } from '../../helpers/dateHelper';
+
+export const FieldsView = ({
+  date,
+  handleSportChange,
+  location,
+  setDate,
+  setTitle,
+  setLocation,
+  setShowDatePicker,
+  showDatePicker,
+  sport,
+  sports,
+  title,
+}) => {
+  const openDatePicker = useCallback(() => {
```

Comment:
The DatePicker is always open now, and can't be dismissed by clicking on cancel or OK

![image](https://user-images.githubusercontent.com/96472119/219652677-7a709e66-e732-4968-b951-a7136e4eb3f1.png)

Reply:
fixed